filters/builtin/builtin.go

```diff
@@ -156,6 +156,8 @@ func Filters() []filters.Spec {
 		NewHeaderToQuery(),
 		NewQueryToHeader(),
 		NewBackendTimeout(),
+		NewReadTimeout(),
+		NewWriteTimeout(),
```

Comment:
We should document these new filters

Reply:
Yes, will do whenever writeTimeout works as I expect :)

Reply:
done